janusgraph-core/src/main/java/org/janusgraph/util/datastructures/RandomRemovalList.java

```diff
@@ -134,7 +134,10 @@ public boolean contains(Object o) {
     public boolean containsAll(Collection<?> c) {
```

Comment:
I would also moved this whole class into `janusgraph-test` because it is used only for tests.

Reply:
I removed the class complete.